src/Microsoft.Fx.Portability.MetadataReader/ReflectionMetadataDependencyInfo.cs

```diff
@@ -196,5 +197,33 @@ public InvalidPEAssemblyException(Exception inner)
                 : base("Not a valid assembly", inner)
             { }
         }
+
```

Comment:
Cool - I'll take a look at that. I would prefer the immutable collections for the guarantees they provide.